api/envoy/config/filter/http/cache/v2/cache.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.cache.v2;
```

Comment:
re: https://github.com/envoyproxy/envoy/pull/7198/files#r369319747 (@mattklein123 @htuch)

I think we can make this v2alpha and v3alpha with current tooling (by leveraging `move_to_package` annotation). So @toddmgreer if this is subject to breaking change, let's do it?

Reply:
Yes, while I don't anticipate needing breaking changes to the config proto, it would be unwise to rule it out. I'll follow whatever the latest consensus is for versioning. I'm also happy to change it post-merge if the preferred versioning continues to evolve.

If so, I presume I'll need to:
1. Change all cache.v[23] in the PR to cache.v[23]alpha.
2. Rename v[23] dir to v[23]alpha dir.
3. Run 'tools/proto_format.sh fix'
4. Confirm that docs/build.sh passes
5. Ask @htuch and/or @lizan to explain the new and exciting errors I discover.

Does that sound about right?

Reply:
Done.